Library/Formula/sdl2.rb

```diff
@@ -26,8 +26,8 @@ class Sdl2 < Formula
   # https://github.com/mistydemeo/tigerbrew/issues/361
   if MacOS.version <= :snow_leopard
     patch do
-      url "https://gist.githubusercontent.com/miniupnp/26d6e967570e5729a757/raw/1a86f3cdfadbd9b74172716abd26114d9cb115d5/SDL2-2.0.3_OSX_104.patch"
-      sha256 "4d01f05f02568e565978308e42e98b4da2b62b1451f71c29d24e11202498837e"
+      url "https://gist.githubusercontent.com/miniupnp/6de808a8939711fcb856eaa049c18883/raw/1e4e0c111fb56c20905a9cd238323a298e1acbe0/SDL2-2.0.8_OSX_105.patch"
```

Comment:
This fails on 10.4, i guess because the patch isn't targeting the older version of OSX :(